nodeup/pkg/model/kubelet.go

```diff
@@ -186,6 +186,8 @@ func (b *KubeletBuilder) buildKubeconfig() (string, error) {
 		return "", fmt.Errorf("error encoding CA certificate: %v", err)
 	}
 
+	cluster.Server = b.Cluster.Spec.MasterInternalName
```

Comment:
Is it imperative that the master use `http://127.0.0.1:8080` or is it ok to use `MasterInternalName` for both the master's and nodes' kubeconfig?